server/controllers/user.js

```diff
@@ -0,0 +1,283 @@
+const jwt = require('jsonwebtoken');
+const models = require('../models');
+const bcrypt = require('bcrypt-nodejs');
+
+const jwtSecret = process.env.JWT_SECRET;
+const User = models.User;
+const emailRegex = /^\w+([.-]?\w+)*@\w+([.-]?\w+)*(\.\w{2,3})+$/;
+
+module.exports = {
+  /**
+   *
+   * Create a user
+   * @param {any} req
+   * @param {any} res
+   * @returns - an object of a created user and a token
+   */
+  createUserWithJwt(req, res) {
+    if (!req.body.fullName) {
+      return res.status(401).json({
+        fullName: 'This Field is Required'
+      });
+    }
+    if (!req.body.userName) {
+      return res.status(401).json({
+        userName: 'This Field is Required'
+      });
+    }
+    if (!req.body.email) {
+      return res.status(401).json({
+        email: 'This Field is Required'
+      });
+    }
+    if (!emailRegex.test(req.body.email)) {
```

Comment:
Consider using the sequelize built-in validator (not Null) rather than manually checking that various fields were specified.